x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/notification/slack/message/SlackMessageTests.java

```diff
@@ -602,6 +602,11 @@ public void testUrlPathIsFiltered() throws Exception {
         }
     }
 
+    public void canHaveNullTextAndAttachment()  throws Exception {
+        SlackMessage slackMessage = new SlackMessage("from", new String[] {"to"}, "icon");
+        assertThat(slackMessage.getText(), is(null));
```

Comment:
This is causing a compile error for me, pls use `assertNull(object)` so it compiles clean

Reply:
Will fix.